indico/web/forms/fields/datetime.py

```diff
@@ -177,6 +177,20 @@ class IndicoTimeField(TimeField):
     widget = JinjaWidget('forms/time_widget.html', single_line=True, single_kwargs=True)
 
 
+class IndicoDurationField(TimeField):
+    widget = JinjaWidget('forms/duration_widget.html', single_line=True, single_kwargs=True)
+
+    def _value(self):
+        if self.data is None:
+            return 0
+        else:
+            return int(self.data.total_seconds())
+
+    def process_formdata(self, valuelist):
+        if valuelist and len(valuelist) == 1:
```

Comment:
I don't think we need the length check here; in the other field we just have it because we need more than one value